.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: python # Set the build language to Python
+
+python: 3.7.1 # Set the version of Python to use
+
+branches: master # Set the branch to build from
+
+install:
+    - pip install mkdocs # Install the required dependencies
+    - pip install mkdocs-material # Install the required dependencies
+
+script:
+    - mkdocs build --verbose --clean # Build a local version of the docs
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+# github-token is Generated in Github Developer Settings -> Personal Access Token (public_repo checked)
+# It also needs to be set in TravisCI as a enviroment variable with the value. (Reminder do not show this in logs.)
+  github-token: $aragon_wiki_github_token  
+  local-dir: site
+  keep-history: true
+  target-branch: gh-pages
+  verbose: true
+  on:
+    branch: master
+
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Comment:
Do we need to specify the distribution and use sudo? Could we get away with not including the matrix?

Reply:
This error comes up when i do not include the matrix on the TravisCI log.

3.7.1 is not installed; attempting download
Downloading archive: https://travis-ci-python-archives.global.ssl.fastly.net/binaries/ubuntu/14.04/x86_64/python-3.7.1.tar.bz2
0.20s$ curl -sSf -o python-3.7.1.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 403 Forbidden
Unable to download 3.7.1 archive. The archive may not exist. Please consider a different version.

Tried on Travis-CI.com and Travis-CI.org.

Will try and see if the build works with python 3.6

Reply:
This is fixed now to use python3.6